go/stream/message_test.go

```diff
@@ -279,4 +281,48 @@ func TestMessage(t *testing.T) {
 		err := validator(&msg)
 		require.EqualError(t, err, "Key: 'Message.Properties.WorkspaceID' Error:Field validation for 'WorkspaceID' failed on the 'required' tag")
 	})
+
+	t.Run("logger fields", func(t *testing.T) {
```

Comment:
We would need to test both with stage webhook and without to test all expectations, right?